cmake/modules/tbb.cmake

```diff
@@ -30,6 +30,7 @@ else()
 				-DBUILD_SHARED_LIBS=Off
 				-DCMAKE_BUILD_TYPE=release
 				-DTBB_OUTPUT_DIR_BASE=lib
+				-DCMAKE_CXX_FLAGS="-Wno-error=stringop-overflow"
```

Comment:
Can we add a little comment / link here to explain the magic? :pray:

Reply:
👍